modules/core/TextureMarble.h

```diff
@@ -0,0 +1,72 @@
+//  Created by Mahmoud El Bergui on 21.04.22.
+//
+
+#pragma once
+#include"Texture.h"
+
+namespace {
```

Comment:
The declaration of the methods in anonymouse namespace is not needed. Just delete this block from the header file.

Reply:
I will proceed in deleting them. I thought it might be nice if a use can see all of the methods on the file